tasks/certificate.yml

```
@@ -6,10 +6,21 @@
 
 - name: Generate new certificate if one doesn't exist
```

```suggestion
- name: "Generate new certificate if one doesn't exist"
```

Quote the name so that vim and possibly other syntax highlighters don't mess with unpaired single quote.